webrecorder/browsertrix
# Initial crawl detail page

WIP https://github.com/webrecorder/browsertrix-cloud/issues/106

Adds new crawl detail page.

TODO in follow-up (all need backend changes):
- File count for completed crawls
- Watch/replay inline or link out

### Manual testing
1. Run app and go to Archives > archive
2. Click a completed crawl. Verify crawl shows with expected data
3. Run and then click running crawl. Verify crawl shows in running state (see screenshot)
4. Cancel and stop crawls. Verify crawl stops as expected

### Screenshots
<img width="1019" alt="Screen Shot 2022-01-29 at 5 17 01 PM" src="https://user-images.githubusercontent.com/4672952/151683177-67206c56-030a-407e-9130-9f34a91dcc1d.png">

<img width="1025" alt="Screen Shot 2022-01-29 at 5 17 10 PM" src="https://user-images.githubusercontent.com/4672952/151683180-f1b86fa4-c61c-49a4-bf73-2569efdca8d6.png">

<img width="328" alt="Screen Shot 2022-01-29 at 5 56 00 PM" src="https://user-images.githubusercontent.com/4672952/151683751-a8509988-5c20-4f1f-a074-cf32000c27dd.png">

<img width="1040" alt="Screen Shot 2022-01-30 at 6 18 01 PM" src="https://user-images.githubusercontent.com/4672952/151730325-43500518-5268-48be-ac71-8f11d198a923.png">
